dandischema/metadata.py

```diff
@@ -101,19 +101,34 @@ def publish_model_schemata(releasedir: str) -> Path:
     return vdir
 
 
+def _validate_obj(data, schema, missing_ok=False):
+    validator = jsonschema.Draft7Validator(
+        schema, format_checker=jsonschema.draft7_format_checker
+    )
+    error_list = []
+    for error in sorted(validator.iter_errors(data), key=str):
+        if missing_ok and "is a required property" in error.message:
+            continue
+        error_list.append([error.message, tuple(error.absolute_path)])
+    if error_list:
+        raise jsonschema.ValidationError(str(error_list))
```

Comment:
this would collapse all errors into one string of a list, making it impossible to print it nicely or "analyze" above.
I guess should be ok for now, but I think we might need to come up with some consistent mechanism between dandischema and dandi-cli to  "encapsulate"  multiple errors as a singular exception which would 
- contain `.errors` list of all individual errors/exceptions (not sure if wrapped into some adapter here to provide consistent interface across errors)
- have nice `__str__` and/or `__repr__`

Reply:
it would be nice to have this. let me look into if python itself has a way of doing this.